### ISSUE TITLE
what is the advantage of fine tune?

### ISSUE THREAD

**Issue by micklexqg** [keras-team/keras#7887]
I am fine-tuning inception v3 (got in keras) with my own dataset, but I find the result is not good. I wonder should I train from scrath rather than fine tune. So, what is the  advantage  and scenarios of fine tune? 
Is there any reference to choose training from scrath or fine-tuning?

**Comment by Mohamedemad4** [comment]
first of this question is better suited for [Cross Validated](https://stats.stackexchange.com) , and to answer your question A lot, I mean you don`t have to train on much data to get good accuracy, you do not have to build a model from scratch  and It saves a ton of headaches from data augmentation to high-end training hardware.
